src/manage_sql.c

```diff
@@ -20964,7 +20964,7 @@ report_add_result (report_t report, result_t result)
      "task_id",                                                              \
      KEYWORD_TYPE_STRING                                                     \
    },                                                                        \
-   { "date", NULL, KEYWORD_TYPE_INTEGER },                                   \
+   { "creation_time", NULL, KEYWORD_TYPE_INTEGER },                          \
```

Comment:
Could you also add "date" as an alias in this line here or as a new column in `REPORT_ITERATOR_WHERE_COLUMNS` and then add it to the filter columns?